src/application.rs

```diff
@@ -190,7 +298,15 @@ impl CoLink {
     }
 
     pub async fn read_entry(&self, key: &str) -> Result<Vec<u8>, Error> {
-        let storage_entry = if key.contains("::") {
+        let storage_entry = if key.contains("$") {
+            // magic mount logic
```

Comment:
Plz feel free to skip the comment if they are obvious - otherwise please make sure you include it everywhere to be consistent. I would highly recommend the former.

Reply:
Sounds good!